service/service.go

```diff
@@ -342,7 +365,7 @@ func (app *Application) shutdownExtensions() {
 	}
 }
 
-func (app *Application) execute() {
+func (app *Application) execute(cfactory ConfigFactory) {
```

Comment:
nit: you use camel-cased name `cFactory` above, please choose a consistent name. Just `factory` might be enough?